frontend/src/styles/modules/_experiment-details.scss

```diff
@@ -251,6 +251,10 @@
       color: $transparent-black-7;
       font-size: $font-unit;
     }
+
+    figcaption img {
```

Comment:
> First, a minor sass syntactical question: why not just put this img block inside the above figcaption block?

Updated!

> Second, why are you setting the width to auto? It looks like it was previously set to 36px.

I was thinking about any future images that wanted to break out of [the width set in `.details-description img`](https://github.com/mozilla/testpilot/blob/master/frontend/src/styles/modules/_experiment-details.scss#L189). Like if someone wanted to add an image in the future to a caption, it could be any width.